cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EndpointPatternDTO.java

```diff
@@ -1,24 +1,37 @@
 package com.cloudesire.platform.apiclient.dto.model.dto;
 
 import com.cloudesire.platform.apiclient.dto.model.enums.EndpointType;
+import io.swagger.annotations.ApiModelProperty;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 import java.util.Objects;
 
-public class EndpointPatternDTO extends NamedEntityDTO
+public class EndpointPatternDTO extends BaseEntityDTO
 {
-    private EndpointType category;
+    @ApiModelProperty( "The URL representation that must contain {nodename} as hostname placeholder" )
+    @NotBlank
+    @Size( max = 4096 )
+    private String name;
+
+    @NotNull
+    private EndpointType category = EndpointType.APP;
 
     private String description;
 
     public EndpointPatternDTO()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)